chartfx-math/src/main/java/de/gsi/math/SimpleDataSetEstimators.java

```diff
@@ -68,7 +113,7 @@ public static double computeInterpolatedFWHM(final double[] data, final int leng
     public static double getDistance(final DataSet dataSet, final int indexMin, final int indexMax,
             final boolean isHorizontal) {
         return isHorizontal ? dataSet.get(DataSet.DIM_X, indexMax) - dataSet.get(DataSet.DIM_X, indexMin)
-                : dataSet.get(DataSet.DIM_Y, indexMax) - dataSet.get(DataSet.DIM_Y, indexMin);
+                            : dataSet.get(DataSet.DIM_Y, indexMax) - dataSet.get(DataSet.DIM_Y, indexMin);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'DataSet.DIM_Y' due to existing static import 'de.gsi.dataset.DataSet.DIM_Y'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)